### PR TITLE
announce HHVM 4.104

### PR DESCRIPTION
**Pull request by jjergus**

### CODE REVIEW

File: _posts/2021-04-05-hhvm-4.104.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+HHVM 4.104 is released! This release marks the end of support for 4.98;
+HHVM 4.99&ndash;4.103 remain supported, as do the 4.56, and 4.80 LTS releases.
+
+Note that HHVM 4.104 is not a long-term support (LTS) release as originally
+scheduled. Instead, HHVM 4.102 was
+[promoted to a LTS release](/blog/2021/03/29/extending-hhvm-4.102-support.html).
+
+# Highlights
+
+- Added support for
```
Comment (fredemmott) [inline _posts/2021-04-05-hhvm-4.104.markdown:17]: These needs postponing, watchman/facts is not currently buildable. Some sources are missing from cmake, and at least macos's compiler doesn't like some of the files

**Review by fredemmott — APPROVED** [review]
remove facts/watchman then done

**Comment by jjergus** [comment]
Thanks. Still waiting for the builds to finish, will commit this as soon as those are done.

File: _posts/2021-04-05-hhvm-4.104.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.104"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.104 is released! This release marks the end of support for 4.98;
+HHVM 4.99&ndash;4.103 remain supported, as do the 4.56, and 4.80 LTS releases.
```
Comment (lexidor) [inline _posts/2021-04-05-hhvm-4.104.markdown:9]: If everything would be going according to the old schedule, 4.56 would be dropped now. If 4.56 is going to be kept for longer than anticipated, that might be something worth noting.

Comment (jjergus) [reply _posts/2021-04-05-hhvm-4.104.markdown:9]: Good catch, you're right, 4.56 should be dropped now.

**Pull request merged**